xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -313,12 +319,13 @@ namespace PVR
    const std::string GetGenresLabel() const;
 
   private:
-    CDateTime    m_recordingTime; /*!< start time of the recording */
+    CDateTime    m_recordingTime;     /*!< start time of the recording */
```

Comment:
Remove the extra blanks after ; , please. This violates our coding guidelines (though used widely, but we should not make it worse.)